### ISSUE TITLE
Packager: Influxdb and telegraf share same init.sh path

### ISSUE THREAD

**Issue by dever860** [influxdb/telegraf#22]
Since influxdb and telegraf share the same ROOT DIR's, the init.sh file will get overwritten when both are installed via RPM.

```
ls -lah /etc/init.d/{telegraf,influxdb}
lrwxrwxrwx 1 root root 21 Jun 22 15:53 /etc/init.d/influxdb -> /opt/influxdb/init.sh
lrwxrwxrwx 1 root root 21 Jun 22 16:06 /etc/init.d/telegraf -> /opt/influxdb/init.sh
```

Suggest changing the following `s/influxdb/telegraf/` so that telegraf is treated independent from influxdb

```
INSTALL_ROOT_DIR=/opt/influxdb
TELEGRAF_LOG_DIR=/var/log/influxdb
CONFIG_ROOT_DIR=/etc/opt/influxdb
```

**Comment by skynet** [comment]
:+1:

**Comment by skynet** [comment]
can someone provide info on how to patch existing influxdb+telegraf installations. I just got my  /opt/influxdb/init.sh overwritten.

**Comment by dever860** [comment]
Ultimately I just did the following, and linked the influxdb init script directly to the absolute URL instead of the symlink in /opt/influxdb.

ln -fs /opt/influxdb/versions/0.9.0/scripts/init.sh /etc/init.d/influxdb

**Comment by skynet** [comment]
I suppose you have to do this, too:

`ln -fs /opt/influxdb/versions/0.1.1/scripts/init.sh /etc/init.d/telegraf`

with this final result:

`
ls -lah /etc/init.d/{telegraf,influxdb}
lrwxrwxrwx 1 root root 44 Jun 25 17:18 /etc/init.d/influxdb -> /opt/influxdb/versions/0.9.0/scripts/init.sh
lrwxrwxrwx 1 root root 44 Jun 25 17:21 /etc/init.d/telegraf -> /opt/influxdb/versions/0.1.1/scripts/init.sh
`

**Comment by toddboom** [comment]
This should be fixed by #23.

**Comment by dever860** [comment]
@toddboom It should be, but #23 was closed and not merged.
